lua/rest-nvim/init.lua

```diff
@@ -260,7 +260,7 @@ local function curl_cmd(opts)
 		if json_body then
 			-- Format JSON output and then add it into the buffer
 			-- line by line because Vim doesn't allow strings with newlines
-			local out = fn.system("echo -E '" .. line .. "' | jq .")
+			local out = fn.system("echo -E '" .. string.gsub(line, "[']", "\\%1") .. "' | jq .")
```

Comment:
Instead of piping into stdin using echo I believe we could make use of the second argument to the system method
```suggestion
			local out = fn.system("jq", line)
```

Reply:
https://neovim.io/doc/user/eval.html#system()

Reply:
well that is cool and it clears both of my recent issues

FWIW, `"\\%1"` wouldn't fix it anyway, but `"'\\%1'"` would

Reply:
Thank you for the suggestion, it makes sense! :)